packages/client/hmi-client/src/services/model.ts

```diff
@@ -141,6 +141,6 @@ export async function generateModelCard(
 	}
 
 	if (modelServiceType === ModelServiceType.TA4) {
-		await modelCard(documentId, modelId);
+		await modelCard(documentId);
```

Comment:
Sometimes you await for modelCard and sometimes you do not await for modelCard - if there is something special that people need to be aware of can you please add some comments to the function declaration?

Reply:
I can add a comment for clarity.  The way its being used right now is to just fire and forget mostly except for that special case where I fetch the model again in the `tera-model-from-code` and `tera-model-from-document-drilldown` components.

Reply:
```
async function generateCard(docId, modelId) {
	if (!docId || !modelId) return;

	if (clonedState.value.modelService === ModelServiceType.TA1 && card.value) {
		return;
	}

	if (clonedState.value.modelService === ModelServiceType.TA4 && goLLMCard.value) {
		return;
	}

	await generateModelCard(docId, modelId, clonedState.value.modelService);
	fetchModel();
}
```